avocado/utils/memory.py

```diff
@@ -265,8 +267,8 @@ def get_buddy_info(chunk_sizes, nodes="all", zones="all"):
 
     if re.findall("[<>=]", chunk_sizes) and buddy_list:
         size_list = range(len(buddy_list[-1][-1].strip().split()))
-        chunk_sizes = [str(_) for _ in size_list if eval("%s %s" % (_,
-                                                                    chunk_sizes))]
+        chunk_sizes = [str(_) for _ in size_list if eval("%s %s" %
+                                                         (_, chunk_sizes))]
```

Comment:
We don't limit the line length to 80 characters in Avocado, but it is a good practice. If it does not compromise the readability, the change is welcomed.